### ISSUE TITLE
ipv6 only not detected

### ISSUE THREAD

**Issue by sputnick-dev** [SirToffski/WireGuard-Ligase#9]
For the need of an IPV6 only VPN, ran `configure-wireguard.sh` script.

Specified `fd42:42:42::1` as my ip and `fd42:42:42::` as my network.

The script seems not intented to support ipv6

**Comment by SirToffski** [comment]
This is correct - I will clarify this in README and add to future “to-do” list.

Thanks!
